awslabs/aws-lambda-powertools-typescript
# build(all): fix vm2 vulnerability

## Description of your changes

Fix vm2 vulnerability (from `aws-cdk`) by running `npm audit fix` in every 

### How to verify this change
1. Run e2e tests to see that there is no broken change.  `aws-cdk` is used in E2E tests for utility packages.
2. Run `npx run cdk synth` on `cdk/example` folder


### Related issues, RFCs
N/A

### PR status

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist


---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
